docs/implementation_plans/2026-02-28_agent_privacy_profiles.md

```
@@ -0,0 +1,811 @@
+    else Legacy path (no profileId anywhere)
+        Workflow->>Resolver: resolveFromModelId(version.modelId ?? template.modelId)
+        Resolver->>Resolver: resolveInferenceProvider(modelId)
+    end
+
+    Resolver-->>Workflow: ResolvedProfile(modelId, provider, ...)
+    Workflow->>Workflow: sendMessage(model: modelId, provider: provider)
+```
+
+### 4.2 Resolution Logic (Pseudocode)
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)